### ISSUE TITLE
Accessibility: 'Buttons do not have an accessible name'

### ISSUE THREAD

**Issue by wildpow** [leandrowd/react-responsive-carousel#266]
Lighthouse accessibility audit message 'Buttons do not have an accessible name'. Here are the two element listed:
```
<button type="button" class="control-arrow control-prev"></button>
<button type="button" class="control-arrow control-next"></button>
```
 Would this pass by just adding a `role` or `aria-label` to these buttons? If so I would like to make a PR for this. I've never contributed to an open source project before and this seems like something easy for a newbie like myself.

**Comment by wildpow** [comment]
Well I have a fork that is passing the Lighthouse accessibility test for these buttons but, now the build is failing 10 tests. What are some first steps I can go about to fix these? I've only just started learning how to use testing frameworks so any resources would be great.
<img width="335" alt="screen shot 2018-10-01 at 5 03 21 pm" src="https://user-images.githubusercontent.com/17464319/46322243-a9415000-c59c-11e8-96ff-62c3eaf6d6e4.png">

**Comment by wildpow** [comment]
PR: https://github.com/leandrowd/react-responsive-carousel/pull/275
